Marlin/src/gcode/config/M43.cpp

```diff
@@ -96,7 +100,11 @@ inline void toggle_pins() {
           watchdog_refresh();
         }
       }
-      pinMode(pin, prior_mode);
+      #ifdef __STM32F1__
+        _SET_MODE(i, prior_mode);
```

Comment:
required instead of pinMode, else give this error: cannot convert 'const gpio_pin_mode' to 'WiringPinMode' for argument '2' to 'void pinMode(uint8, WiringPinMode)'

Reply:
This could be `_SET_MODE` on all platforms if that's compatible in all cases.

Reply:
Ah, I see it is not. Good enough.